lasagne/objectives.py

```diff
@@ -341,45 +342,50 @@ def multiclass_hinge_loss(predictions, targets, delta=1):
 def huber_loss(predictions, targets, delta=1):
     """ Computes the huber loss between predictions and targets.
 
-    .. math:: L_\delta (diff) = \frac{diff^2}{2} & \text{if  } |diff| \le \
-                                \delta
-    .. math:: L_\delta (diff) & = & \delta (|diff| - \frac{\delta}{2} ),\
+    .. math:: 
+        L_\delta (diff) = \frac{diff^2}{2} & \text{if  } |diff| \le \ delta \\
+
+        L_\delta (diff) & = & \delta (|diff| - \frac{\delta}{2} ),\ \\
```

Comment:
The alignment characters (`&`) still seem off -- does this render correctly in Sphinx?